data/countries/PK.yaml

```diff
@@ -0,0 +1,46 @@
+holidays:
+  PK:
+    names:
+      en: Pakistan
+    langs: []
```

Comment:
Could you please add the languages used in this data?
e.g.
- en
- ur (Urdu)
It would be nice if you could also add local languages if available e.g. Urdu